kubernetes/config-maps/nginx-proxy/default.conf.j2

```diff
@@ -5,22 +5,22 @@ map $http_accept $extension {
     application/json json;
 }
 
-{% if nginx_proxy_ssl %}
-server {
+{% if not nginx_proxy_allow_http %}
+ server {
```

Comment:
Could you merge the `server` blocks and put all conditions inside it. I think it would make the config a little simpler.

Reply:
I think we can do something like this: https://serverfault.com/a/474345, but I don't know, if this solution will simplify config.

Reply:
OK, nevermind. It can stay as is. I thought that we can join the blocks without adding the extra `if`.